src/mainwindow.cpp

```diff
@@ -1104,6 +1106,16 @@ void MainWindow::initializeSettingsDatabase()
         updateFrame();
         m_settingsDatabase->setValue(QStringLiteral("splitterSizes"), m_splitter->saveState());
     }
+
+    // isNoteListCollapsed
+    if (m_settingsDatabase->value(QStringLiteral("isNoteListCollapsed"), "NULL") == "NULL") {
+        m_settingsDatabase->setValue(QStringLiteral("isNoteListCollapsed"), false);
+    }
+
+    // isTreeCollapsed
+    if (m_settingsDatabase->value(QStringLiteral("isTreeCollapsed"), "NULL") == "NULL") {
+        m_settingsDatabase->setValue(QStringLiteral("isTreeCollapsed"), false);
+    }
```

Comment:
I think we can just remove this. For bool values, `false` is the default anyway, so there should be no point in inserting false for initialization.